pulp_rpm/app/migrations/0026_rpmrepository_last_sync_repomd_cheksum.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.16 on 2020-10-15 11:55
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('rpm', '0025_remove_orphaned_subrepos'),
```

Comment:
Please rebase and fix the migration number and dependency, we merged some recently.